cmd/sonobuoy/app/status.go

```diff
@@ -26,15 +26,14 @@ import (
 	"github.com/spf13/cobra"
 	"k8s.io/client-go/kubernetes"
 
-	"github.com/heptio/sonobuoy/cmd/sonobuoy/app/args"
 	ops "github.com/heptio/sonobuoy/cmd/sonobuoy/app/operations"
 	"github.com/heptio/sonobuoy/pkg/errlog"
 )
 
-var statusOpts struct {
-	namespace string
-	config    args.Kubeconfig
-}
+var (
```

Comment:
what do we think about this? The advantage of the anonymous struct is it namespaces these variables.

Reply:
I don't have strong opinions outside of the client interface.  

Once it gets to the client we will likely want to wrap in a struct as signatures have a tendency to change over time.